This is modeled after the ia64 support. Aarch64 can be either big endian
or little endian, so we add the case for both.

---
 native/fdlibm/ieeefp.h | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/native/fdlibm/ieeefp.h b/native/fdlibm/ieeefp.h
index 1a9740f..73455c0 100644
--- a/native/fdlibm/ieeefp.h
+++ b/native/fdlibm/ieeefp.h
@@ -27,6 +27,14 @@
 #endif
 #endif
 
+#ifdef __aarch64__
+#ifdef __BIG_ENDIAN__
+#define __IEEE_BIG_ENDIAN
+#else
+#define __IEEE_LITTLE_ENDIAN
+#endif
+#endif
+
 #ifdef __hppa__
 #define __IEEE_BIG_ENDIAN
 #endif
-- 
2.16.3

